atk/src/auto/document.rs

```diff
@@ -119,6 +130,7 @@ impl<O: IsA<Document>> DocumentExt for O {
         }
     }
 
+    #[doc(alias = "load-stopped")]
```

Comment:
dash?

Reply:
Yeah, we discussed this before. See comments above. This is intended to help people find the methods from the name of the signal from C doc.